R/parse.R

```diff
@@ -10,6 +10,38 @@ parse_package <- function(base_path, load_code, registry, global_options = list(
   list(env = env, blocks = blocks)
 }
 
+#' Parse roxygen2 comments in a piece of code.
+#'
+#' [parse_code()] takes a file or character vector, and an environment with R
+#' values, and parses the roxygen2 comments found in it.
+#'
+#' @param file Path of the file to process.
+#' @param env An environment containing the result of parsing and evaluating
+#'   the file.
+#' @param registry A roclet tag registry.
+#' @param wrap Logical; should roxygen2 output be wrapped? `FALSE` by default.
+#' @param markdown Logical value indicating whether to parse Markdown tags.
+#'   This can be overridden locally by the tags '@rd` and `@NoRd` inside a
+#'   Roxygen comment.
+#' @param file_encoding The file encoding. Default: `"UTF-8"`.
+#' @param text Instead of specifying `file`, users can also specify a character
+#' vector `text`, containing the R code.
```

Comment:
Needs indent.  Maybe document with text?

Can you please add `@keyword internal`